### ISSUE TITLE
Priority 1 accessibility issue: The captions presented within a video are incorrect

### ISSUE THREAD

**Issue by alicerichmond** [wellcomecollection/wellcomecollection.org#6544]
**What**

The captions presented within the video are incorrect. Additionally, the captions presented as an option within the YouTube video are obscured due to the transparency and text appearing behind them.

Code Ref(s): #\39 14291272
 <iframe title="Video" allowfullscreen="" src="https://www.youtube.com/embed/w6XvC1bWeuc?feature=oembed?rel=0&amp;enablejsap i=1" frameborder="0" class="iframe-container__iframe absolute"></iframe>

Solution:
Please ensure that correct captions are available which can be read clearly.
           

**Where**

URL: https://wellcomecollection.org/events/YAcEiRAAACQAx1CP Page: Intensive Care with Dr Gavin Francis | Wellcome Collection

**Comment by dannybirchall** [comment]
This should be discussed with Thea Stevenson. 'Incorrect' isn't specified, but this is live captioning, so the subtitles lag behind the speaker.

I'm not sure how / whether this can be corrected.

The YT captions could simply be turned off to avoid the clash.

**Comment by pollecuttn** [comment]
https://wellcome.slack.com/archives/C3N7J05TK/p1625656260067200:

"The videos having subtitles issue was actually because the events were live-captioned at the time of recording - so that was why there were some errors and delays, because that's a natural part of live-captioning, and it also means the subtitles cannot be removed because a) they're built in and b) they should be because they're an authentic part of how the event was on the day it was live. I think we discussed putting a sentence on the page to explain that the event had been live-captioned at the time and that the live captioning was on the recording, so there was context on the page to make sense of this for people who encountered it?"

Still to be done by @dannybirchall 's team

**Comment by jennpb** [comment]
No further action required
